### PR TITLE
[ResourceAction] LPS-117531 read portlet resource per deployment after portlet model initialized

### PR DESCRIPTION
**Pull request by dantewang**
This is about reading portlet resource actions after portlet model is prepared. The other pull (Spring extender related) failed some tests on CI, will send tomorrow.

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test

**Comment by tinatian** [comment]
@dantewang This pull is incomplete to me, I will wait for your another pull to review all together.

File: portal-kernel/src/com/liferay/portal/kernel/security/permission/ResourceActionsUtil.java [hunk 1]
```diff
@@ -277,6 +277,24 @@ public static boolean isRootModelResource(String modelResource) {
 		return getResourceActions().isRootModelResource(modelResource);
 	}
 
+	public static void read(
```
Comment (tinatian) [inline portal-kernel/src/com/liferay/portal/kernel/security/permission/ResourceActionsUtil.java:280]: @dantewang @yuhai Why we need this version of API? You have already the one with multiple arguments.


Tina.

Comment (yuhai) [reply portal-kernel/src/com/liferay/portal/kernel/security/permission/ResourceActionsUtil.java:280]: @tinatian, I will remove the api.

**Pull request closed without merging**